src/connectionController.ts

```diff
@@ -551,7 +559,7 @@ export default class ConnectionController {
       throw connectError;
     }
 
-    log.info('Successfully connected');
+    log.info('Successfully connected', { connectionId });
```

Comment:
Adding the connection name as well might help debugging as it can be hard to infer which connection the connectionId belongs to.

Reply:
We don't pass the connection name anywhere outside the connection controller, not sure we can use it to match connectivity issues here and for example in LS. But I added a name to the explorer logs, so we can find the info what was expanded:
https://github.com/mongodb-js/vscode/pull/563/files#diff-06a415e2a54193ec0955688552bf1cb4813ac5868fc9392083f3c37f8f65a8c4R67-R69